docs/tutorials/excel-tutorial.md

```diff
@@ -1007,3 +969,4 @@ In this tutorial, you've created an Excel task pane add-in that interacts with t
 - [Office Add-ins platform overview](../overview/office-add-ins.md)
```

Comment:
nit: In step 7 of the [Open the dialog from the task pane](https://review.learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial?branch=pr-en-us-3801#open-the-dialog-from-the-task-pane) section, the TODO1 comment should end with a period.